example_simulation/example.conf

```diff
@@ -0,0 +1,58 @@
+[conf_1]
+material = [gold_fcc]
+# Arg: Mandatory
+# Type: List or string
+# Discrp: Materials to be simulated. If using custom materials, the names shall
+#         correspond to atoms object(s) in the python file(s). If using materials project,
+#         use the prefix "mp_" followed by the formula to be searched for.
+materials_path = /path/to/example_simulation/[materials] 
+# Arg:    Optional
+# Type:   String
+# Discrp: The path where the materials are stored. Note: change the
+#         '/path/to/' to the selected path directory. Otherwise the 'startup' command 
+#         won't work. [materials] can be either a folder with python files or a python file.
+workspace_path = /path/to/example_workspace 
+# Arg:    Mandatory
+# Type:   String
+# Discrp: The selected path of the workspace. Note: if no example_workspace exist,
+#         a example_workspace will be created.
+ensemble = [NVT]
+# Arg:    Mandatory
+# Type:   String or list
+# Discrp: The ensemble/ensembles for the selected materials. If single then string, if
+#         many then a list.
+potential = [openkim]
+# Arg:    Mandatory
+# Type:   String or list
+# Discrp: Type of potential to be used on the materials. 
+kim-model = EAM_Dynamo_Ackland_1987_Au__MO_754413982908_00
+# Arg:    Mandatory if potential is openkim, otherwise optional.
+# Type:   String
+# Discrp: The string is selected from openkim. This string is generated by selecting
+#         the specific material in the periodic table from: https://openkim.org/
+initial-temperature = 40 
+# Arg:    Mandatory
+# Type:   Float
+# Discrp: The intial temprature of the simulation.
+target-temperature = 40 
+# Arg:    Mandatory if ensemble is NVT, otherwise optional
+# Type:   Float
+# Discrp: The intial temprature of the simulation.
+repeat = 0 
+# Arg:    Mandatory
+# Type:   Int
+# Discrp: The cellsize (generated by repeated sized of a unitcell) of the simulation.
+#         Example, if repeat=0 then simulating a unitcell.
+steps = 100 
+# Arg:    Mandatory
+# Type:   Int
+# Discrp: Number of timesteps for the simulation, 1 timestep=1fs
+use-asap = False
+# Arg:    Mandatory
+# Type:   Boolean
+# Discrp: True if the user want to use Asap, otherwise False
+volume-scale = (a,b,c) 
+# Arg:    Optional
+# Type:   Tupple, a,b: float and c: int

```

Comment:
it's not tuple, it's a syntax for specifying a range.
it can also be single value e.g. "1" or "1.3"
or it can be a list "[1, 2, 3, 4]"
but the example with a, b, c can stay since that's what we will mainly use.

Reply:
done